### PR TITLE
GEN-2901: Addons UI

### PR DESCRIPTION
**Pull request by juliaendre**
## [GEN-2901]

- Added Addons as a new module
- Added design according to this [Figma](https://www.figma.com/design/SNVLvLq7ztclXiDqqMNXgY/Tiers-%26-Addons-(App)?node-id=2765-167390&node-type=canvas&t=5WsfPuZG8vJy81Ar-0)
- Under unleash feature flag. Not sure how to place that feature flag yet.

Notes
- Localization will be added when we have final design

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Home/Sources/Models/QuickAction.swift [hunk 1]
```diff
@@ -32,6 +33,8 @@ public enum QuickAction: Codable, Equatable, Hashable, Sendable {
             return L10n.hcQuickActionsUpgradeCoverageTitle
         case .cancellation:
             return L10n.hcQuickActionsTerminationTitle
+        case .addon:
+            return "Edit addon"
```
Comment (sladan-hedvig) [inline Projects/Home/Sources/Models/QuickAction.swift:37]: I guess its fine until we have an entry point(s)

File: Projects/hCore/Sources/Models/EditType.swift [hunk 2]
```diff
@@ -3,13 +3,15 @@ public enum EditType: String, Codable, Hashable, CaseIterable {
...
     case cancellation
+    case addon
 
     public var title: String {
         switch self {
         case .coInsured: return L10n.contractEditCoinsured
         case .changeAddress: return L10n.InsuranceDetails.changeAddressButton
         case .changeTier: return L10n.InsuranceDetails.changeCoverage
         case .cancellation: return L10n.hcQuickActionsCancellationTitle
+        case .addon: return "Addon"
```
Comment (sladan-hedvig) [inline Projects/hCore/Sources/Models/EditType.swift:14]: I guess its fine until we decide on entry point(s)

File: Projects/Addons/Sources/Views/ChangeAddonScreen.swift [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+                hText(subTitle, style: .label)
+                    .foregroundColor(hTextColor.Opaque.secondary)
+            }
+
+            if changeAddonVm.selectedAddonOptionId == addonOption.id.uuidString && changeAddonVm.hasSubOptions {
+
+                let colorScheme: ColorScheme = UITraitCollection.current.userInterfaceStyle == .light ? .light : .dark
+                DropdownView(
+                    value: String(changeAddonVm.getSelectedSubOption?.subtitle ?? "") + " dagar",
+                    placeHolder: "Välj skydd"
```
[the remark this answers is not in the document]
Comment (juliaendre) [reply Projects/Addons/Sources/Views/ChangeAddonScreen.swift]: @sladan-hedvig are you referring to "Välj skydd"? why do we want it there?

File: Projects/Addons/Sources/Views/ChangeAddonSummaryScreen.swift [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                    documents: self.contractInformation?.documents ?? [],
+                    onDocumentTap: { document in
+
+                    },
+                    displayItems: self.contractInformation?.displayItems ?? [],
+                    insuranceLimits: self.contractInformation?.insurableLimits ?? [],
+                    typeOfContract: self.contractInformation?.typeOfContract
+                )
+            ],
+            total: .init(amount: "220", currency: "SEK")
```
[the remark this answers is not in the document]
Comment (juliaendre) [reply Projects/Addons/Sources/Views/ChangeAddonSummaryScreen.swift:41]: I guess we will get this from backend

**Review by sladan-hedvig — CHANGES_REQUESTED** [review]

**Review by sladan-hedvig — APPROVED** [review]

**Pull request merged**